src/ResourceManagement/WebSite/Microsoft.Azure.Management.Websites/Generated/Models/Address.cs

```diff
@@ -29,7 +28,13 @@ public partial class Address
         /// <summary>
         /// Initializes a new instance of the Address class.
         /// </summary>
-        public Address(string address1 = default(string), string address2 = default(string), string city = default(string), string country = default(string), string postalCode = default(string), string state = default(string))
+        /// <param name="address1">Address 1.</param>
+        /// <param name="city">City.</param>
+        /// <param name="country">Country.</param>
+        /// <param name="postalCode">Postal code.</param>
+        /// <param name="state">State.</param>
+        /// <param name="address2">Address 2.</param>
+        public Address(string address1, string city, string country, string postalCode, string state, string address2 = default(string))
```

Comment:
We changed some of these parameters to be required since our REST API actually rejects null values here